nist_800_171_rev_2/si.sp

```diff
@@ -44,6 +44,7 @@ benchmark "nist_800_171_rev_2_3_14_3" {
   description = "There are many publicly available sources of system security alerts and advisories. For example, the Department of Homeland Security’s Cybersecurity and Infrastructure Security Agency (CISA) generates security alerts and advisories to maintain situational awareness across the federal government and in nonfederal organizations. Software vendors, subscription services, and industry information sharing and analysis centers (ISACs) may also provide security alerts and advisories. Examples of response actions include notifying relevant external organizations, for example, external mission/business partners, supply chain partners, external service providers, and peer or supporting organizations."
   children = [
     control.guardduty_enabled,
+    control.redshift_cluster_maintenance_settings_check,
```

Comment:
please add the `nist_800_171_rev_2  = "true"` tag to `redshift_cluster_maintenance_settings_check` control.